src/drivers/esp32/src/esp_rom_bootloader.c

```diff
@@ -37,3 +37,17 @@
 #include "task.h"
 #include "uart2.h"
 
+void espRomBootloaderInit()
```

Comment:
Maybe it would be nice wit a comment explaining the sequence below ... not sure though, maybe it is self-explanatory?